pkgs/data/fonts/junicode/default.nix

```
@@ -1,20 +1,21 @@
 { lib, fetchzip }:
 
 fetchzip {
-  name = "junicode-0.7.8";
+  name = "junicode-1.002";
```

In the spirit of https://github.com/NixOS/rfcs/pull/35:

```suggestion
  pname = "junicode";
  version = "1.002";
```

---

I just realized that this is using `fetchzip`, not `mkDerivation`. I'm not sure if the "name from pname and version" behaviour is implemented here. If it isn't, you additionally need to set

```
name = "${pname}-${version}";
```

---

Not only doesn't it support ‘name from pname and version’, it actually gives me an error about unexpected argument ‘version’ in fetchurl if I try. So I let-bound them instead (following the example of [agave](https://github.com/NixOS/nixpkgs/blob/260761649bd857ea73eeb6cd21c7e6c9830d7215/pkgs/data/fonts/agave/default.nix#L3).